### PR TITLE
239: add uselang as optional parameter for wbsearchentities to specif…

### PR DESCRIPTION
**Pull request by Lhaaits**
…y required response language

### CODE REVIEW

**Comment by Lhaaits** [comment]
Resolves #239

File: wdtk-wikibaseapi/src/main/java/org/wikidata/wdtk/wikibaseapi/WbSearchEntitiesAction.java [hunk 1]
```diff
@@ -69,7 +69,17 @@ public WbSearchEntitiesAction(ApiConnection connection, String siteUri) {
...
-				properties.strictlanguage, properties.type, properties.limit, properties.offset);
+				properties.strictlanguage, properties.type, properties.limit, properties.offset, properties.uselang);
+	}
+
+	/**
+	 * Keeping this for backwards compatibility, the real action happens in 
+	 * {@link WbSearchEntitiesAction#wbSearchEntities(String, String, Boolean, String, Long, Long, String)}
+	 */
+	public List<WbSearchEntitiesResult> wbSearchEntities(String search, String language,
+														 Boolean strictLanguage, String type, Long limit, Long offset)
```
Comment (wetneb) [inline wdtk-wikibaseapi/src/main/java/org/wikidata/wdtk/wikibaseapi/WbSearchEntitiesAction.java:80]: Funny indentation here (and below too). I guess I need to get on with the linting setup (we have something that works quite well in OpenRefine).

**Review by wetneb — APPROVED** [review]

File: wdtk-wikibaseapi/src/main/java/org/wikidata/wdtk/wikibaseapi/WbSearchEntitiesAction.java (hunk 1 above)
Comment (Lhaaits) [reply wdtk-wikibaseapi/src/main/java/org/wikidata/wdtk/wikibaseapi/WbSearchEntitiesAction.java]: Yeah, the intelliJ autoformat and the format used here do not get along well

**Comment by Lhaaits** [comment]
Yes please :)

**Comment by wetneb** [comment]
done :)

**Pull request merged**